### PR TITLE
Fixed feedback hover

### PR DESCRIPTION
**Pull request by MitchelPaulin**
Fixed both the jittering and highlighting issue with the feedback

temp6

### CODE REVIEW

File: src/components/rubric-feedback.js [hunk 1]
```diff
@@ -91,13 +91,11 @@ export class RubricFeedback extends composeMixins(
...
 
 				.feedback-arrow {
-					margin-top: calc(-0.5rem - 42px);
+					margin-top: calc(-0.5rem - 43px);
 					width: 0;
 					height: 0;
 					border: 12px solid transparent;
 					position: absolute;
-					transition: border-color 0.5s;
-					transition-timing-function: ease;
```
Comment (MitchelPaulin) [inline src/components/rubric-feedback.js:100]: Not sure about this one, removed the transition so its consistent with the rest of the outline, I can add the transition to both if we want it

File: src/components/rubric-feedback.js [hunk 2]
```diff
@@ -573,8 +556,6 @@ export class RubricFeedback extends composeMixins(
 				<div
 					class="feedback-wrapper"
 					?data-desktop="${!this.compact}"
-					@mouseover="${this._addFocusStylingToFeedbackWrapper}"
-					@mouseout="${this._removeFocusStylingFromFeedbackWrapper}"
```
Comment (MitchelPaulin) [inline src/components/rubric-feedback.js:577]: No longer needs to be handled by the feedback

File: src/components/rubric-feedback.js (hunk 1 above)
Comment (imcdonald-d2l) [reply src/components/rubric-feedback.js:100]: makes sense to keep in consistent in my opinion 👍🏻

**Review by imcdonald-d2l — APPROVED** [review]

**Comment by MitchelPaulin** [comment]
Vdiff changes were just from the 1px buffer I added to prevent the movement during hovering, each image was just `1px * numberOfFeedbackCells` larger

**Pull request merged**